src/main/java/com/seatapp/services/LoggerService.java

```diff
@@ -0,0 +1,31 @@
+package com.seatapp.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoggerService {
```

Comment:
Debug in logger is not enabled at the moment, either enable debug logging or remove te loggers.

Reply:
The loggers are now enabled.